language_detector/main.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    globalcount = 0
+    lang = ""
+    
+    for dictionary in languages:
+        
+        count = 0
+        for word in text.split():
```

Comment:
Some suggestion: 
- remove the unnecessary symbols, such as `{}[]()<>` etc 
- convert the words in lowercase for more efficient search. Since: 

```python
>>>"this" == "This"
False
```